app/ClubEvent.php

```diff
@@ -38,15 +53,21 @@ class ClubEvent extends Model
 								'evnt_title',
 								'evnt_subtitle',
 								'plc_id',
-								'evnt_show_to_club',
 								'evnt_date_start',
 								'evnt_date_end',
 								'evnt_time_start',
 								'evnt_time_end',
 								'evnt_public_info',
 								'evnt_private_details',
 								'evnt_is_private',
-								'evnt_is_published');
+								'evnt_is_published',
+                                'external_id',
+                                'price_tickets_normal',
+                                'price_tickets_external',
+                                'price_normal',
+                                'price_external',
+                                'facebook_done',
+                                'event_url');
```

Comment:
Looks like indentation is broken her?

Reply:
Which editor are you guys using and why does it mix tabs and spaces? There has to be a setting to fix that!?

Reply:
i'm using intellij with the standard formatting rulesset for php
if this is such big problem you can provide a editorconfig, most ide's will respect this
http://editorconfig.org/